bot_file.py

```diff
@@ -339,7 +339,9 @@ def known_user_functions(self):
             'report-responses' : self.report_responses,
             'rollbar-item' : self.rollbar_item,
             'elm-progress' : self.elm_progress,
-            'elm-progress-on' : self.elm_progress_on
+            'elm-progress-on' : self.elm_progress_on,
+            'find-017-matches' : self.find_elm_017_matches,
+            'how-hard-to-port' : self.how_trivial_to_port
```

Comment:
:art: :art: how-hard-to-port is how-trivial-to-port :) either rename the function to `how_hard_to_port`, or add a doc comment to `how_trivial_to_port` about the scale of trivialness (smaller the more trivial)?